specification/search/resource-manager/Microsoft.Search/preview/2020-08-01-preview/search.json

```diff
@@ -1158,7 +1158,7 @@
         "tags": [
           "Services"
         ],
-        "description": "Checks whether or not the given Search service name is available for use. Search service names must be globally unique since they are part of the service URI (https://<name>.search.windows.net).",
+        "description": "Checks whether or not the given search service name is available for use. search service names must be globally unique since they are part of the service URI (https://<name>.search.windows.net).",
```

Comment:
Again, we need to leave this second instance of "search" capitalized since it's the beginning of a sentence.